scripts/fast-tune.sh

```
@@ -56,12 +56,82 @@ case "$APP" in
     FUNCTESTS=example/postgres-vuln-functional-tests.yaml
     PROFILE_MATCH=replicaset-pg-vuln
     ;;
+  argocd-server)
+    NS=argocd
+    SUITE=example/argocd-server-attacks.yaml
+    FUNCTESTS=example/argocd-server-functional-tests.yaml
+    PROFILE_MATCH=argocd-server-.*-argocd-server
```

Shellcheck (0.11.0)

[warning] Brace expansions and globs are literal in assignments. Quote it or use an array.

(SC2125)